Sources/Testing/Testing.docc/Expectations.md

```diff
@@ -10,15 +10,54 @@ See https://swift.org/LICENSE.txt for license information
 See https://swift.org/CONTRIBUTORS.txt for Swift project authors
 -->
 
-Check for expected values and outcomes in tests.
+Check for expected values, outcomes, and asynchronous events in tests.
 
 ## Overview
 
-The testing library provides `#expect()` and `#require()` macros you use to 
-validate expected outcomes. To validate that an error is thrown, or _not_ thrown, 
-the testing library provides several overloads of the macros that you can use.
-Use a ``Confirmation`` to confirm the occurrence of an asynchronous event that
-you can't check directly using an expectation.
+Use ``expect(_:_:sourceLocation:)`` and
+``require(_:_:sourceLocation:)-5l63q`` macros to validate expected
+outcomes. To validate that an error is thrown, or _not_ thrown, the
+testing library provides several overloads of the macros that you can
```

Comment:
These overloads aren't discussed in this file. Did you want to link out to them?

Reply:
Yes, I'll also link to the confirmation article as that isn't discussed here either.